static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -17,6 +17,7 @@ import type {
 import { Advert } from 'commercial/modules/dfp/Advert';
 import { breakpointNameToAttribute } from 'commercial/modules/dfp/breakpoint-name-to-attribute';
 import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
+import 'prebid.js/build/dist/prebid';
```

Comment:
Does this replace the `require.ensure` to tell webpack to bundle this module?

Reply:
yes